File: 2.0/java/orders/orderKeywords.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.common.model.ResourceId;
import com.callfire.api.client.api.keywords.model.request.KeywordPurchaseRequest;

import java.util.Arrays;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        KeywordPurchaseRequest request = KeywordPurchaseRequest.create()
            .keywords(Arrays.asList("SUN", "MOON")).build();
        ResourceId resourceId = callfireClient.ordersApi().orderKeywords(request);
    }
}

File: 2.0/csharp/orders/orderKeywords.cs
using CallfireApiClient;
using CallfireApiClient.Api.Common.Model;
using CallfireApiClient.Api.Keywords.Model.Request;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new KeywordPurchaseRequest {Keywords = {"SUN", "MOON"}};
        ResourceId resourceId = client.OrdersApi.OrderKeywords(request);
    }
}
